tds.go

```diff
@@ -965,6 +966,27 @@ func parseConnectParams(dsn string) (connectParams, error) {
 		}
 	}
 
+	// https://msdn.microsoft.com/en-us/library/dd304019.aspx
+	// https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlconnection.connectionstring.aspx
+	// https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/configure-the-network-packet-size-server-configuration-option
```

Comment:
This link is great. However it seems to point out that the max size of an encrypted connection 16383 bytes. We should either limit it at that or detect if the connection is unencrypted for the 32767 byte size. This is fine to be here but it should be presented with a full sentence saying what it is for. I'd also like this link in the user visible documentation (the README), again with it in a context in a sentence. 

"For more information when to set package size see the docs on (MSDN)[link here]".